internal/pkg/cli/app_deploy_test.go

```diff
@@ -284,7 +284,7 @@ func TestAppDeployOpts_getAppDockerfilePath(t *testing.T) {
 
 	mockError := errors.New("mockError")
 	mockManifest := []byte(`name: appA
-type: 'Load Balanced Web App'
+type: 'Load Balanced Web Svc'
```

Comment:
I think the customer visible strings should be "Service" instead of "Svc" to make it obvious to them